docs/developers/extending.rst

```diff
@@ -82,7 +82,7 @@ Configurable options
 Extensions cannot require modifications to the Django settings of Open Forms. Any run-time configuration option can
 be specified as:
 
-#. Deployment time environment variables. See `plugin.py:35 <https://github.com/open-formulieren/demo-extension/blob/main/demo_extension/plugin.py#L35>`_.
+#. Deployment time environment variables. See `plugin.py:35 <https://github.com/open-formulieren/demo-extension/blob/main/demo_extension/plugin.py>`_.
```

Comment:
How come are these documentation changes + `ci.yaml` changes part of this PR?

Reply:
> How come are these documentation changes + `ci.yaml` changes part of this PR?

I think I did something wrong with the git rebase